pkg/detectors/easyinsight/easyinsight.go

```diff
@@ -60,27 +64,37 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 			}
 
 			if verify {
-				data := fmt.Sprintf("%s:%s", resIdMatch, resMatch)
-				sEnc := b64.StdEncoding.EncodeToString([]byte(data))
+				auth := fmt.Sprintf("%s:%s", resIdMatch, resMatch)
+				sEnc := b64.StdEncoding.EncodeToString([]byte(auth))
+
```

Comment:
**Nit**: Moving the verification code will improve the resources management (memory clean-up etc) and code readability.